main.js

```diff
@@ -11,3 +11,140 @@ function closemenu() {
 //  binding the event to HTML
 menu.addEventListener('click', openmenu);
 menu.addEventListener('click', closemenu);
+
+const workInfo = [
+  {
+    Id: '1',
+    name: 'Tonic',
+    img: 'snapshoot-portfolio.png',
+    descriptions: 'hfjfjhfconsectetur adipisicing elit. Animi quae quos dolor numquam, quibusdam tempora illum nulla facere sunt deleniti?',
```

Comment:
- [ ] nice work. For now, all your popups are showing the same thing (the first array). Please kindly ensure it iterates through the arrays so each popup represents a project of its own. This will be advantageous when you add your projects to it. Nice work.